veros/setup/acc_sector/acc_sector.py

```diff
@@ -0,0 +1,184 @@
+     - basic usage of diagnostics
+
+    :doc:`Adapted from ACC channel model </reference/setups/acc>`.
+
+    """
+    
+    max_depth = 4000.
+    so_wind_factor = 1.
+
+    @veros.veros_method
+    def set_parameter(self):
+        self.identifier = "acc_sector"
+
+        self.nx, self.ny, self.nz = 15, 62, 40
+        self.dt_mom = 3600.
+        self.dt_tracer = 3600.
+        self.runlen = 86400 * 365
+
+        self.coord_degree = True
+        self.enable_cyclic_x = True
+
+        self.congr_epsilon = 1e-12
+        self.congr_max_iterations = 5000
+
+        self.enable_neutral_diffusion = True
+        self.K_iso_0 = 1000.0
+        self.K_iso_steep = 500.0
+        self.iso_dslope = 0.005
+        self.iso_slopec = 0.01
+        self.enable_skew_diffusion = True
+
+        self.enable_hor_friction = True
+        self.A_h = 5e4 * 2 
+        self.enable_hor_friction_cos_scaling = True
+        self.hor_friction_cosPower = 1
+
+        self.enable_bottom_friction = True
+        self.r_bot = 1e-5
+
+        self.enable_implicit_vert_friction = True
+
+        self.enable_tke = True
+        self.c_k = 0.1
+        self.c_eps = 0.7
+        self.alpha_tke = 30.0
+        self.mxl_min = 1e-8
+        self.tke_mxl_choice = 2
+        self.kappaM_min = 2e-4
+        # self.enable_tke_superbee_advection = True
+        self.enable_Prandtl_tke = False
+
+        self.K_gm_0 = 1300.0
+        self.enable_eke = False
+        self.eke_k_max = 1e4
+        self.eke_c_k = 0.4
+        self.eke_c_eps = 0.5
+        self.eke_cross = 2.
+        self.eke_crhin = 1.0
+        self.eke_lmin = 100.0
+        self.enable_eke_superbee_advection = False
+        self.enable_eke_isopycnal_diffusion = False
+
+        self.enable_idemix = False
+        self.enable_idemix_hor_diffusion = False
+        self.enable_eke_diss_surfbot = False
+        self.eke_diss_surfbot_frac = 0.2
+        self.enable_idemix_superbee_advection = False
+
+        self.eq_of_state_type = 3
+
+    @veros.veros_method
+    def set_grid(self):
+        self.dxt[...] = 2.0
+        self.dyt[...] = 2.0
+        self.x_origin = 0.0
+        self.y_origin = -60.0
+        self.dzt[...] = veros.tools.get_vinokur_grid_steps(self.nz, self.max_depth, 10., refine_towards="lower")
+
+    @veros.veros_method
+    def set_coriolis(self):
+        self.coriolis_t[:, :] = 2 * self.omega * np.sin(self.yt[None, :] / 180. * self.pi)
+
+    @veros.veros_method
+    def set_topography(self):
+        x, y = np.meshgrid(self.xt, self.yt, indexing="ij")
+        self.kbot = np.logical_or( (x > 1.0) & (x < 27), y < -40).astype(np.int)
+
+        # A half depth (ridge) is appended to the domain within the confines
+        # of the circumpolar channel at the inflow and outflow regions
+        bathymetry = np.logical_or( ((x <= 1.0) & (y < -40)), ((x >= 27) & (y < -40)) )
+        kzt2000 = np.sum((self.zt < -2000.).astype(np.int))
+        self.kbot[bathymetry] *= kzt2000
```

Comment:
Why multiplication? Shouldn't this be `self.kbot[bathymetry] = kzt2000`?

Reply:
You're right!